src/core/js/scriptLoader.js

```diff
@@ -30,7 +30,8 @@
                 a11y: 'libraries/jquery.a11y',
                 scrollTo: 'libraries/scrollTo',
                 bowser: 'libraries/bowser',
-                'enum': 'libraries/enum'
+                enum: 'libraries/enum',
+                jqueryMobile: 'libraries/jquery.mobile.custom'
```

Comment:
It's a fair point. It would eliminate the need for code like this:
```
onSwipe: function(e) {
    // Don't allow swiping with the mouse
    if (!Adapt.device.touch) {
         return;
     }
     doSomething();
}
```